404.html

```diff
@@ -2,67 +2,76 @@
 <html>
 
 <head>
-  <title>Page not found</title>
+  <title>Error</title>
```

Comment:
Currently, to preserve region/locale information in the URL for the 404 page, we need to duplicate it across multiple regions. This can become cumbersome if authors need to update the 404 page in the future.

Ideally, we should maintain a single 404 file and configure CDN-level rules to retain the URL structure while serving content from the same file.